Sources/FileSystem/FileSystemTestingTrait.swift

```diff
@@ -0,0 +1,32 @@
+#if DEBUG
+    #if canImport(Testing)
```

Comment:
```suggestion
#if DEBUG && canImport(Testing)
```

Correct me if I'm wrong but I believe you should be able to combine the compiler directives into a single statement to lower the amount of nesting needed